Mathlib/Combinatorics/SimpleGraph/Subgraph.lean

```diff
@@ -1174,6 +1174,12 @@ theorem deleteVerts_mono {G' G'' : G.Subgraph} (h : G' ≤ G'') :
     G'.deleteVerts s ≤ G''.deleteVerts s :=
   induce_mono h (Set.diff_subset_diff_left h.1)
 
+@[mono]
+lemma deleteVerts_mono' {G' : SimpleGraph V} (u : Set V) (h : G ≤ G') :
+    ((⊤ : Subgraph G).deleteVerts u).coe ≤ ((⊤ : Subgraph G').deleteVerts u).coe := by
```

Comment:
@b-mehta, thoughts?

Reply:
Oops - didn't see this till now. This looks pretty unpleasant: I'm surprised you need something of this form... But let's go with it for now.